### ISSUE TITLE
Invalid commands in yggdrasilctl return a generic error

### ISSUE THREAD

**Issue by vpzomtrrfrt** [yggdrasil-network/yggdrasil-go#385]
```
$ sudo yggdrasilctl fsadkfjsk
Admin socket returned an error but didn't specify any error text
```

**Comment by neilalexander** [comment]
Yep, this should be really simple to improve. I think there might be a few cases where the admin socket isn't quite as verbose as it could be in the case of error.

**Comment by neilalexander** [comment]
Fixed by #395.
